source/dem/insertion.cc

```diff
@@ -74,17 +74,17 @@ Insertion<dim>::assign_particle_properties(
         -particle_sizes[particle_counter];
       double density =
         physical_properties.density[current_inserting_particle_type];
-      double vel_x = 0.;
-      double vel_y = 0.;
-      double vel_z = 0.;
-      double w_x   = 0.;
-      double w_y   = 0.;
-      double w_z   = 0.;
-      double mass  = density * (1.3333 * M_PI * (diameter * 0.5) *
+      double vel_x   = 0.;
+      double vel_y   = 0.;
+      double vel_z   = 0.;
+      double omega_x = 0.;
+      double omega_y = 0.;
+      double omega_z = 0.;
+      double mass    = density * (1.3333 * M_PI * (diameter * 0.5) *
```

Comment:
What is the 1.333? Is it the 4/3? Why not write it as 4./3.? It would be less confusing.

Reply:
Done.